app/views/layouts/application.html.erb

```diff
@@ -12,10 +12,10 @@
         <ul class="navigation">
-            <li class="links signup"><a href="#signup">Sign up</a></li>
-            <li class="links signin"><a href="file:///Users/Mbarona/Documents/signin.html">Sign in</a></li>
+            <li class="links signup"><a href="/users/sign_up">Sign up</a></li>
```

Comment:
use the named routes for devise here.

Reply:
@micronix Named routes to Devise? Link_to, new_user_session...? 

I had tried doing so but the application kept crashing.

Reply:
what was the error it gave you? Also looks like there is a merge conflict related to Mins latest push. Can you pull the changes from upstream and resolve the merge conflict.